clarity_epp/export/illumina.py

```diff
@@ -238,19 +245,22 @@ def get_project(projects, urgent=False):
                 override_cycles=';'.join(override_cycles)
             )
 
-    for line in lims.get_file_contents(id=file_id).rstrip().split('\n'):
-        if line.startswith('[Settings]') and custom_settings:
-            output_file.write('{line}\n'.format(line=line))
+    for data in csv.reader(
+        lims.get_file_contents(id=file_id).rstrip().split('\n'),
+        quotechar='"', delimiter=',', quoting=csv.QUOTE_ALL, skipinitialspace=True
+    ):
+        if data[0] == '[Settings]' and custom_settings:
```

Comment:
Are there any samplesheet parser libraries publicly available? If so, would be nice to use instead of these checks whether the line is header, which section you are etc. 

Probably those libraries can also validate a samplesheets format.

Reply:
I am not aware of any library (that would fit our criteria), but last time I searched is a long time ago.